onnxruntime/python/tools/kernel_explorer/kernels/fast_gelu_test.py

```diff
@@ -0,0 +1,100 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import sys
+
+sys.path.append("../build")
+
+import kernel_explorer as ke
+import numpy as np
+import pytest
+
+
+def dtype_to_funcs(dtype):
+    type_map = {
+        "float16": list(filter(lambda x: "FastGelu_half" in x, dir(ke))),
+        "float32": list(filter(lambda x: "FastGelu_float" in x, dir(ke))),
+        "float64": list(filter(lambda x: "FastGelu_double" in x, dir(ke))),
+    }
+    return type_map[dtype]
+
+
+def fast_gelu(x, bias):
+    x = x + bias
+    y = 0.5 * x * (1 + np.tanh(0.797885 * x + 0.035677 * x * x * x))
+    return y
+
+
+@pytest.mark.skip(reason="called by test_fast_gelu_all_sizes")
```

Comment:
maybe change the function name to run_fast_gelu. In that way, we do not need skip test here.

Reply:
Done.